daprdocs/content/en/reference/components-reference/supported-pubsub/setup-rabbitmq.md

```diff
@@ -90,6 +92,7 @@ The above example uses secrets as plain strings. It is recommended to use a secr
 | maxLenBytes      | N        | Maximum length in bytes of a queue and its dead letter queue (if dead letter enabled). If both `maxLen` and `maxLenBytes` are set then both will apply; whichever limit is hit first will be enforced.  Defaults to no limit. | `"1048576"` |
 | exchangeKind      | N        | Exchange kind of the rabbitmq exchange.  Defaults to `"fanout"`. | `"fanout"`,`"topic"` |
 | saslExternal      | N        | With TLS, should the username be taken from an additional field (for example, CN). See [RabbitMQ Authentication Mechanisms](https://www.rabbitmq.com/access-control.html#mechanisms).  Defaults to `"false"`. | `"true"`, `"false"` |
+| ttlInSeconds      | N        | Set TTL at the component or message level. Message level TTL overwrites the default component level TTL. | `"60"` |
```

Comment:
This explains the component level ttl metadata.
My try: Set message TTL at the component level, which can be overwritten by message level TTL per request.